workorders.lic

```diff
@@ -280,6 +280,10 @@ class WorkOrders
       end
 
       wait_for_script_to_complete('shape', ['log', recipe['chapter'], recipe['name'], info['stock-name'], recipe['noun']])
+      log_num = bput('read my log', 'You must bundle and deliver \d+ more').scan(/\d+/).first.to_i
+      if log_num == quantity
+        exit
```

Comment:
Does `workorders` need to do any cleanup like lines 289 or 290 before it exits?

Reply:
I wonder if this should `break` instead of `exit` and then `workorders` will clean up as it normally does? I'm not sure what other side effects that would have.